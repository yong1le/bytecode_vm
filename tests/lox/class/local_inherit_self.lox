{
  class Foo < Foo {} // expect compile error: A class can't inherit from itself.
}
// [c line 5] Error at end: Expect '}' after block.
