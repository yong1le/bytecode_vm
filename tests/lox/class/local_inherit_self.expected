[line 2]: Error at 'Foo': A class cannot inherit from itself.